manifest.versions.json

```diff
@@ -1,13 +1,13 @@
 {
     "variables": {
-      "kb|ltsc2016|3.5-4.7.2":"kb5014702",
-      "lcu|ltsc2016|3.5-4.7.2":"https://catalog.s.download.windowsupdate.com/c/msdownload/update/software/secu/2022/06/windows10.0-kb5014702-x64_48f484f06070f3dee90aa62a574d9c1e1b774b2a.msu",
+      "kb|ltsc2016|3.5-4.7.2":"kb5015808",
+      "lcu|ltsc2016|3.5-4.7.2":"https://catalog.s.download.windowsupdate.com/c/msdownload/update/software/secu/2022/07/windows10.0-kb5015808-x64_eb7156d8c49c8f23fe3a3f0a3f18b827eeae4530.msu",
       "kb|ltsc2019|3.5-4.7.2": "kb5013892",
```

Comment:
This one isn't updated because it's based on a .NET Fx LCU, not Windows, and .NET Fx wasn't serviced this month.